views/shared/_dc_footer.ejs

```diff
@@ -3,7 +3,7 @@
   <span class="dc-footer--text">
     Put your R skills to the test
   </span>
-  <a class="dc-footer--link btn btn-primary" href="https://www.datacamp.com/signal">Start Now</a>
+  <a class="dc-footer--link btn btn-primary" href="https://assessment.datacamp.com/r-programming">Start Now</a>
```

Comment:
Would you want some more snowplow tracking out of the box? Could be nice to add a specific additional class name like `ds-snowplow-btn-rdocs-signal` or so

Reply:
Good idea, also added a few `utm_*` query params, I believe snowplow should pick these up too right?

Reply:
Yes! utm parameters are picked up too and have a ton of flexibility!